commit 263a260c9bb7af53afc7abcebb00fac8273c1c6b
Author: WANG Xuerui <git@xen0n.name>
Date:   Thu Jun 25 21:33:31 2020 +0800

    [stage2] gdbm: turn off -fno-common to fix build on gcc-10

diff --git a/PKGBUILD b/PKGBUILD
index 6366b50..eab7c20 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -18,12 +18,14 @@ md5sums=('988dc82182121c7570e0cb8b4fcd5415'
 
 build() {
   cd $pkgname-$pkgver
-  ./configure --prefix=/usr --enable-libgdbm-compat
+  export CFLAGS="$CFLAGS -fcommon"
+  ./configure --host=@CHOST@ --build=@BUILDHOST@ --prefix=/usr --enable-libgdbm-compat
   make
 }
 
 check() {
   cd $pkgname-$pkgver
+  export CFLAGS="$CFLAGS -fcommon"
   make check
 }
 
